pkg/ha/ha_parser.go

```diff
@@ -0,0 +1,100 @@
+package ha
```

Comment:
The file name is misleading. We do not implement any language parser. We just check validity of labels and parsing data. `parse_ts.go` seems easier.